tests/Services/TwilioTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ivan
+ * Date: 05.06.16
+ * Time: 17:56
+ */
+
+namespace tests\Services;
+
+use App\Services\Twilio;
+use TestCase;
+use Services_Twilio;
+use Services_Twilio_Rest_Call;
+
+class TwilioTest extends TestCase
+{
+
+    public function testFindCall()
+    {
+        /*
+        $client = $this->getMockBuilder(Services_Twilio::class)
+            ->setConstructorArgs(self::getCredentials())
+            ->setMethods(['account'])
+            ->getMock();
+
+        $response = new stdClass();
+        $response->calls = new stdClass();
+
+
+        $client->expects($this->once())
+            ->method('account')
+            ->willReturn($response);
+*/
+        $twilio = $this->getTwilioService();
+        $result = $twilio->findCall('Test');
+
+        $this->assertInstanceOf(Services_Twilio_Rest_Call::class, $result);
```

Comment:
Actually, I wanted to test method findCall. I thought that Twillio api will return null if there is no call with such id and I will assert if object is instance of Services_Twilio_Rest_Call, in this case method works as expected, either test will not pass. But as I realised it always return instance of Services_Twilio_Rest_Call (I didn't find somewhere in documentation description about it), but this object will be empty if there is no calls.
I understand that I have no time to finish tests, I stoped at this moment and send my code to Jasper